tape/src/main/java/com/squareup/tape/QueueFile.java

```diff
@@ -457,6 +467,68 @@ public synchronized int forEach(ElementVisitor reader) throws IOException {
     return elementCount;
   }
 
+  @Override public Iterator<byte[]> iterator() {
+    return new ElementIterator();
+  }
+
+  private final class ElementIterator implements Iterator<byte[]> {
```

Comment:
Fixed, I have a PR coming up later for other places we can cut out synthetic accessor methods too.